website/docs/prepare-for-merge.md

```diff
@@ -0,0 +1,261 @@
+---
+id: prepare-for-merge
+title: Prepare for The Merge
+sidebar_label: Prepare for The Merge
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::caution DRAFT
+
+**This document is being actively developed** and may change as we receive feedback from users like you. Join our [Discord server](https://discord.gg/prysmaticlabs) to share your feedback.
+
+:::
+
+
+[The Merge](https://ethereum.org/en/upgrades/merge/) will fully transition Ethereum's consensus mechanism from proof-of-work to proof-of-stake. This is made possible by the [Beacon Chain](https://ethereum.org/en/upgrades/beacon-chain/), a new Ethereum network layer that implements proof-of-stake consensus. After The Merge, this consensus layer will be fully "merged" with Ethereum's execution layer, and miners will be fully replaced by validators on Ethereum Mainnet.
+
+This guide will walk you through the changes that you need to make in preparation for The Merge.
+
+:::info New user?
+
+This guidance is targeted at users who are already running Prysm. If you're starting from scratch, our [Quickstart](./install/install-with-script.md) will help you configure a Merge-ready node from start to finish.
+
+:::
+
+## The Merge: Before and after
+
+| Before The Merge                                                                           | After The Merge                                                                                 |
+|--------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------|
+| You don't need to run a local execution client. You can use a service like Infura instead. | You **do** need to run an execution client. You **can't** use a service like Infura.            |
+| Execution clients don't need to use the engine API.                                        | Execution clients **do** need to use the engine API.                                            |
```

Comment:
Internal detail not sure if we should mention this in a table geared towards end-users